src/index.css

```diff
@@ -46,10 +49,36 @@ img {
 
 .container {
   padding: 2px 16px;
-    display: flex;
-    flex-direction: row;
-    flex-wrap: wrap;
-    justify-content: center;
-    width: auto;
-    align-items: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: auto;
+  align-items: center;
+}
+.HeaderBar {
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: space-between;
+  padding: 0px 47px;
+  height: 50px;
+  align-items: center;
+  background: #3f51b5;
```

Comment:
take these colors as common as a theme so you can change them anytime in future.